python/pyspark/sql/streaming.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, jsqm):
     def active(self):
         """Returns a list of active queries associated with this SQLContext
 
+        >>> sdf = spark.readStream.format('text').load('python/test_support/sql/streaming')
```

Comment:
I think we might be able define a global path in `_test()` and reuse the variable for`'python/test_support/sql/streaming'` because I guess that path does not really mean anything to users.

I (personally) prefer to follow the reference `numpy` here but if there is no case for this, then we could define a global variable and reuse it.